numpy/ma/core.py

```diff
@@ -3875,6 +3875,12 @@ def compressed(self):
         >>> type(x.compressed())
         <class 'numpy.ndarray'>
 
+        >>> x = np.ma.array(np.arange(9).reshape(3, 3), mask=[[1, 0, 0],
+                                                              [1,0,0],
+                                                              [0,0,0]])
```

Comment:
```suggestion
        ...                                                   [1, 0, 0],
        ...                                                   [0, 0, 0]])
```
Not sure how we format the new line, but this should be OK.  You do need the `...` to continue the input line though, otherwise it looks like a single line with output (and the test fails).